CustomApps/lyrics-plus/OptionsMenu.js

```diff
@@ -111,7 +113,7 @@ const TranslationMenu = react.memo(({ showTranslationButton, translatorLoaded, i
 				Spicetify.ReactComponent.Menu,
 				{},
 				react.createElement("h3", null, " Conversions"),
-				translatorLoaded || !isJapanese
+				translatorLoaded || isJapanese
```

Comment:
I think that using `translatorLoaded || !isJapanese` is better. Because I add netease music translation in this custom app. When spotify play an English songs, the translator won't loaded the Japanese translator, however it may load Chinese translation from netease music. So it should loaded the menu option immediately.